2018-golab/MiningPatterns.md

```diff
@@ -0,0 +1,17 @@
+# Mining GoLang patterns
+
+**Updates:** https://github.com/src-d/conferences/issues/33
+
+**Sent on:** not sent
+
+**Status:** thinking
+
+**Author:** Vadim? Francesc?
+
+**Abstract:**
+
+I thought that since we've got our deduplication engine and many famous Go repositories, it will be possible to find the functions or snippets which are widely repeated throughout codebases. So we could talk about the results and how we made it.
```

Comment:
I was thinking about something similar but finding duplicates with project code and standard library
Basically a "don't reinvent the wheel as a service"

Reply:
goodgopher? hehe